maven-plugin-report-plugin/src/main/java/org/apache/maven/plugin/plugin/report/PluginReport.java

```diff
@@ -93,6 +110,9 @@ public class PluginReport extends AbstractMavenReport {
     @Parameter
     private List<RequirementsHistory> requirementsHistories = new ArrayList<>();
 
+    @Parameter(defaultValue = "[0,)")
+    private String detectRequirementsHistory;
```

Comment:
The naming includes to be boolean, but is string. Weird.

Reply:
yes, it looked strange, but I did not precisely see why :)
renamed to `requirementsHistoryDetectRange`